assets/js/components/HostDetails/HostSummary.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import ListView from '@components/ListView';
+import HealthIcon from '@components/Health/HealthIcon';
+import Tooltip from '@components/Tooltip';
+import ClusterLink from '@components/ClusterLink';
+
+const prepareTooltipContent = (ipAddresses) => (
+  <div className="grid grid-cols-2 gap-1">
+    {ipAddresses.map((ip, index) => (
+      <div key={ip}>
+        {ip}
+        {index % 2 === 0 && ipAddresses.length - 1 !== index && ' '}
+      </div>
+    ))}
+  </div>
+);
+
+const renderIpAddresses = (ipAddresses) => {
+  const joinedIpAddresses = ipAddresses.join(', ');
+  if (ipAddresses.length < 3) {
+    return <span>{joinedIpAddresses}</span>;
+  }
+
+  return (
+    <div className="flex flex-row">
+      <Tooltip
+        className="grid grid-flow-row grid-cols-1 grid-rows-1"
```

Comment:
Do we need all classnames? I think the result would be the same removing all of them